<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fred</class>
 <widget class="QDialog" name="fred">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>1071</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>831</width>
     <height>1071</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>280</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Connecter</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>196</x>
       <y>180</y>
       <width>207</width>
       <height>56</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>ID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ID"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Mot de passe</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="MotDePasse"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QPushButton" name="Gestion_Agents">
     <property name="geometry">
      <rect>
       <x>254</x>
       <y>110</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Gestion des agents</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Gestion_Conges">
     <property name="geometry">
      <rect>
       <x>254</x>
       <y>190</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Gestion des conges</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QTableView" name="TabAgents">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>130</y>
       <width>431</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="Ajouter_Agents">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>420</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Modifier_Agents">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>420</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Supprimer_Agents">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>420</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Afficher_Agents">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>420</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Afficher</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Retour_Agents">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>40</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>allez au conges</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>26</x>
       <y>130</y>
       <width>221</width>
       <height>251</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>ID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ID_Agents"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Nom</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Nom_Agents"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Prenom</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Prenom_Agents"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Mot de passe</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Motdepasse_Agents">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Adresse</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Adresse_Agents"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Role</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="Role_Agents">
          <item>
           <property name="text">
            <string>Administrateur</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Employe</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QPushButton" name="Retour_Conges">
     <property name="geometry">
      <rect>
       <x>614</x>
       <y>40</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>allez au agents</string>
     </property>
    </widget>
    <widget class="QTableView" name="Tab_Conges">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>160</y>
       <width>431</width>
       <height>261</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>450</y>
       <width>392</width>
       <height>30</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_18">
      <item>
       <widget class="QPushButton" name="Ajouter_Conges">
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Modifier_Conges">
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Supprimer_Conges">
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Afficher_Conges">
        <property name="text">
         <string>Afficher</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>271</width>
       <height>221</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_19">
        <item>
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>Type de conge</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="Type_Conge">
          <item>
           <property name="text">
            <string>Maladie</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Conge de maternite</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Conge chomepaye</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_20">
        <item>
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>Date de conge</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="Date_Conge"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_21">
        <item>
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>Date de fin conge</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="Date_Fin_Conge"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
